src/main/java/ovh/spajstech/azure/rest/EmployeeController.java

```diff
@@ -0,0 +1,41 @@
+package ovh.spajstech.azure.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import ovh.spajstech.azure.repository.EmployeeRepository;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class EmployeeController {
+    @Autowired
+    private EmployeeRepository repository;
+
+    @GetMapping("/api/employees/")
```

Comment:
Remove slash to avoid 405:
```suggestion
    @GetMapping("/api/employees")
```